admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -34,33 +34,28 @@ export default class OrderProductRenderer {
     this.router = new Router();
   }
 
-  removeProductFromList(orderDetailId) {
-    const $productRow = $(OrderViewPageMap.productsTableRow(orderDetailId));
-    $productRow.hide('fast', () => $productRow.remove());
-
-    const numProducts = parseInt($(OrderViewPageMap.productsCount).html(), 10);
-    $(OrderViewPageMap.productsCount).html(numProducts - 1);
-  }
-
   addOrUpdateProductToList(orderProductId, newRow) {
     const $productRow = $(OrderViewPageMap.productsTableRow(orderProductId));
     if ($productRow.length > 0) {
       $productRow.html($(newRow).html());
     } else {
       $(OrderViewPageMap.productAddRow).before($(newRow).hide().fadeIn());
     }
+  }
 
-    const numProducts = parseInt($(OrderViewPageMap.productsCount).html(), 10);
-    $(OrderViewPageMap.productsCount).html(numProducts + 1);
+  updateNumProducts(numProducts) {
+    $(OrderViewPageMap.productsCount).html(numProducts);
   }
 
-  editProductFromToList(orderProductId, quantity, priceTaxIncl, priceTaxExcl, taxRate) {
+  editProductFromToList(orderProductId, quantity, priceTaxIncl, priceTaxExcl, taxRate, location, availableQuantity) {
```

Comment:
I did not see where this function is called

Reply:
admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js l.75